nncf/experimental/tensor/functions/numeric.py

```diff
@@ -381,6 +381,20 @@ def round(a: Tensor, decimals=0) -> Tensor:
     return Tensor(round(a.data, decimals))
 
 
+@functools.singledispatch
+@tensor_guard
+def power(a: Tensor, exponent: float) -> Tensor:
```

Comment:
```suggestion
def power(a: Tensor, exponent: Union[Tensor, float]) -> Tensor:
```

https://numpy.org/doc/stable/reference/generated/numpy.power.html

Reply:
I've updated test cases and add tensor notation